package.json

```diff
@@ -36,7 +36,6 @@
     "webpack-stream": "3.1.0"
   },
   "devDependencies": {
-    "browser-sync": "2.11.2",
```

Comment:
Yeah! Just check output on Travis. I moved them over one-by-one when `devDependencies` was originally added and for some reason Travis didn't like them all moved over. It would be great to get at the underlying reason why though